src-tauri/src/main.rs

```diff
@@ -516,17 +516,19 @@ async fn main() {
             });
 
             let command_clone = strm3.clone();
-            let app_clone = app_.clone();
+            let app_next = app_.clone();
+            let app_previous = app_.clone();
+            let app_toggle = app_.clone();
```

Comment:
If you have any idea how to avoid cloning these, please let me know! I've been driving myself crazy with the borrow checker trying to get this working, especially within those async closures inside of other closures.

Reply:
No, this is my project with rust or svelte...